self_driving/nvidia_prediction.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from self_driving.simulation_data import SimulationDataRecord
+from self_driving.image_processing import preprocess
+
+MAX_SPEED = 25
+MIN_SPEED = 10
+
+class NvidiaPrediction:
+    def __init__(self, model):
+        self.model = model
+        self.speed_limit = MAX_SPEED
+
+    def predict(self, image, car_state: SimulationDataRecord):
+        try:
+            image = np.asarray(image)
+
+            image = preprocess(image)
+            image = np.array([image])
+
+            steering_angle = float(self.model.predict(image, batch_size=1))
+
+            speed = car_state.vel_kmh
+            if speed > self.speed_limit:
+                self.speed_limit = MIN_SPEED  # slow down
+            else:
+                self.speed_limit = MAX_SPEED
+            throttle = 1.0 - steering_angle ** 2 - (speed / self.speed_limit) ** 2
```

Comment:
@p1ndsvin Do we configure the `speed_limit` using the input parameter passed to `competition.py`?

Reply:
the parameter can substitute MAX_SPEED